src/main/java/org/jenkinsci/maven/plugins/hpi/TestInsertionMojo.java

```diff
@@ -67,7 +67,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             w.println("/**");
             w.println(" * Entry point to auto-generated tests (generated by maven-hpi-plugin).");
             w.println(" * If this fails to compile, you are probably using Hudson &lt; 1.327. If so, disable");
-            w.println(" * this code generation by configuring maven-hpi-plugin to &lt;disabledTestInjection>true&lt;/disabledTestInjection>.");
+            w.println(" * this code generation by configuring maven-hpi-plugin to &lt;disabledTestInjection;&gt;true&lt;/disabledTestInjection&gt;.");
```

Comment:
it is probably easier to just use `{@code <disabledTestInjection>true</disabledTestInjection>}` as you do not need to escape `<` or `>` in `{@code }` blocks.  It is certainly easier to read if you end up looking at the source code rather than the javadoc!